website/src/components/SearchPage/fields/DateField.tsx

```diff
@@ -67,16 +67,18 @@ const CustomizedDatePicker: FC<CustomizedDatePickerProps> = ({
     valueToDateConverter,
     fieldValue,
 }) => {
+    const dateValue = fieldValue !== '' ? valueToDateConverter(fieldValue.toString()) : null;
     return (
         <div>
             <div className='flex justify-between items-center'>
                 <label htmlFor={field.name} className='block text-sm w-16 my-3 text-right mr-2 text-gray-400'>
                     {field.label}
                 </label>
                 <DatePicker
+                    value={dateValue}
                     name={field.name}
-                    defaultValue={fieldValue !== '' ? valueToDateConverter(fieldValue.toString()) : undefined}
```

Comment:
Interesting - we had to add this here: https://github.com/loculus-project/loculus/pull/1991 but now we don't. Maybe it's `null` vs `undefined` that is the key.